CHANGELOG.md

```diff
@@ -14,6 +14,18 @@ Releases
 This high level changelog is usually updated when a release is tagged.
 On the master branch there may be changes that are not (yet) described here.
 
+### 1.15.0
+
+* [-] fix issue sending pin codes
+* [+] add command line option to bind to specific interface
+* [+] Use ifferent hints for OTP and 2FA
+* [-] fix coverity defects
+* [-] always define user agent
+* [+] remove password from /proc/#/cmd
+* [+] extend OTP to allow FTM push
+* [+] add support for host checks
+* [-] don't accept route to the vpn gateway
+
```

Comment:
maybe https://github.com/adrienverge/openfortivpn/commit/7dd50633efab167da0a29b63d6b37725627f6b80 is worth to be mentioned. The description from the commit message, however, should be rephrased here, maybe something like 'fix byte counter in pppd_write'